stackslib/src/net/chat.rs

```diff
@@ -1195,6 +1195,7 @@ impl ConversationP2P {
         network: &mut PeerNetwork,
         message: &mut StacksMessage,
         authenticated: bool,
+        ibd: bool,
```

Comment:
This seems like it should be either (1) some kind of global, or (2) a state variable of `PeerNetwork` or `ConversationP2P`, rather than a function argument.

Reply:
It gets passed in from the node implementation, since it is able to determine whether or not there are more blocks to process based on burnchain state.  I'd prefer to leave it this way.

Reply:
Do we have a consensus? Is PeerNetwork created from the node implementation? Cause in that case, making if a state variable of PeerNetwork would be what I would lean towards, but I am happy to do whatever.

Reply:
It needs to be an argument to `PeerNetwork::run()`, since `ibd` is computed from state that lives exclusively in the node implementation.  Please leave as-is.